### ISSUE TITLE
11th Voice not working ??

### ISSUE THREAD

**Issue by nfroeschl** [ElectroTechnique/TSynth-Teensy4.1#107]
Today I tried the Master-MT branch (9298d69) and I just played 1 note several tie and I realized that every 11th note was missing. I see on the voice grid that it is always the same voice (11th) which is missing. Could someone try if they have the same problem with this build ?

**Comment by ElectroTechnique** [comment]
Yes, it's been noticed. There are one or two bugs still. Unison & Monophonic causes crashes for instance.

**Comment by ElectroTechnique** [comment]
Unison & Monophonic causing crashes is now fixed in V2.31. Haven't had the 11th (and 9th and 10th) voice missing problem in a while. A bit strange, it seemed to be caused by plugging into USB after being unplugged for a while?!?

**Comment by glido1** [comment]
Contact tsynth with USB also causes DAW crashes, e.g. Cubase, and if it does not crash, the CPU load reaches 90%. I tested on win7 64 with cubase 5 and win 10 with cubase 10, After several hours of searching for the problem, I disconnected the USB and no more i use.
The Midi goes through the Motu interface and all TSYNTHs are plugged into the Motu I2S bus.
Two months have passed and I have no problems.
Additionally, I can use the 32 bit DSP database offered by MOTU.
